examples/hello_world/module/rust/src/lib.rs

```diff
@@ -26,146 +26,100 @@ pub mod proto {
 
 #[derive(Default)]
 struct Main;
 
-oak::entrypoint_command_handler!(oak_main => Main);
-
 impl oak::CommandHandler for Main {
     type Command = ConfigMap;
 
     fn handle_command(&mut self, _command: Self::Command) -> anyhow::Result<()> {
         let log_sender = oak::logger::create()?;
         oak::logger::init(log_sender.clone(), log::Level::Debug)?;
-        let translator_sender_option =
-            oak::io::entrypoint_node_create::<translator_common::TranslatorEntrypoint, _, _>(
-                "translator",
-                &Label::public_untrusted(),
-                "translator",
-                LogInit {
-                    log_sender: Some(log_sender.clone()),
-                },
-            )
-            .ok();
-        if translator_sender_option == None {
-            log::warn!("could not create translator node");
-        }
-        let handler_sender = oak::io::entrypoint_node_create::<Handler, _, _>(
-            "handler",
+
+        let router_sender = oak::io::entrypoint_node_create::<Router, _, _>(
+            "router",
             &Label::public_untrusted(),
             "app",
-            Init {
+            LogInit {
                 log_sender: Some(log_sender),
-                translator_sender: translator_sender_option,
             },
         )?;
-        oak::grpc::server::init_with_sender("[::]:8080", handler_sender)?;
+        oak::grpc::server::init_with_sender("[::]:8080", router_sender)?;
         Ok(())
     }
 }
 
-struct Handler {
-    translator: Option<translator_common::TranslatorClient>,
+oak::entrypoint_command_handler_init!(router => Router);
+
+#[derive(Default)]
+pub struct Router {
+    /// Log sender channel to be sent to every newly created Handler Node.
+    log_sender: Sender<LogMessage>,
 }
 
-impl oak::WithInit for Handler {
-    type Init = Init;
+impl oak::WithInit for Router {
+    type Init = LogInit;
 
     fn create(init: Self::Init) -> Self {
-        oak::logger::init(init.log_sender.unwrap(), log::Level::Debug).unwrap();
-        Self {
-            translator: init
-                .translator_sender
-                .map(translator_common::TranslatorClient),
-        }
+        let log_sender = init.log_sender.unwrap();
+        oak::logger::init(log_sender.clone(), log::Level::Debug).unwrap();
+        Self { log_sender }
     }
 }
 
-oak::entrypoint_command_handler_init!(handler => Handler);
-oak::impl_dispatcher!(impl Handler : HelloWorldDispatcher);
+/// Creates individual Handler Nodes for each client request and assigns client labels to them.
+impl CommandHandler for Router {
+    type Command = grpc::Invocation;
 
-impl Handler {
-    fn translate(&self, text: &str, from_lang: &str, to_lang: &str) -> Option<String> {
-        let client = self.translator.as_ref()?;
-        translator_common::translate(client, text, from_lang, to_lang)
-    }
-}
+    fn handle_command(&mut self, invocation: Self::Command) -> anyhow::Result<()> {
```

Comment:
Perhaps add a brief comment here. Does it create two nodes with the same label?

Reply:
Comment was added above `impl`
https://github.com/project-oak/oak/pull/1758/files#diff-68dc93d24ee4fa90a8be1ebc02c4b778eabcc2ae2898d5e6de9fa7773f313b35R85